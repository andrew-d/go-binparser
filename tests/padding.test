[
    {"name": "PaddingTest",
     "fields": [
        {"name": "first", "offset": 3, "type": "uint32"},
        {"name": "second", "offset": 15, "type": "uint32"}
    ]}
]
