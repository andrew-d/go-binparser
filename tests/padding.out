First,%!s(uint32=0)
Second,%!s(uint32=0)
